specs/wp-signup-spec.js

```diff
@@ -505,38 +431,27 @@ testDescribe( `[${host}] Sign Up  (${screenSize}, ${locale})`, function() {
 								} );
 							} );
 
-							test.it( 'The sign up processing page will finish and show a \'Continue\' button', function() {
-								return this.signupProcessingPage.waitForContinueButtonToBeEnabled();
-							} );
-
-							test.it( 'Clicking the \'Continue\' button continues the process', function() {
-								return this.signupProcessingPage.continueAlong();
+							test.it( 'Can then see the sign up processing page and it will finish and show a \'Continue\' button, which is clicked', function() {
```

Comment:
With this change it looks like you can remove lines 427-432 above (starting with `test.it( 'Can then see the sign up processing page', function() {` ).

Reply:
Yes, good catch 👍